toolflow/vivado/platform/netfpga_sume/board.xdc

```diff
@@ -20,6 +20,8 @@
 #The following two properties should be set for every design
 set_property CFGBVS GND [current_design]
 set_property CONFIG_VOLTAGE 1.8 [current_design]
+set_property BITSTREAM.GENERAL.COMPRESS TRUE [current_design]
+set_property BITSTREAM.CONFIG.UNUSEDPIN Pullup [current_design]
```

Comment:
Is the pullup safe? What is the reason for it?

Reply:
That comes from the reference XDC, so I hope it is safe... Had some stability issues with one of my boards and tried to rule out floating pins.

Reply:
From the reference XDC seems good :+1: